packages/components/psammead-radio-schedule/src/ProgramCard/index.jsx

```diff
@@ -128,20 +133,21 @@ const programStateConfig = {
 const renderHeaderContent = ({
   state,
   link,
-  stateLabel,
+  nextLabel,
+  liveLabel,
   brandTitle,
   episodeTitle,
   service,
   script,
   startTime,
   timezone,
   locale,
+  dir,
 }) => {
-  const isOnDemand = state === 'onDemand';
   const isLive = state === 'live';
-  const hiddenTextProps = stateLabel === 'Live' ? { lang: 'en-GB' } : {};
+  const isNext = state === 'next';
```

Comment:
This update reduces the logic here and makes this much cleaner.

In future, I'd rather we pull out this Live/Next logic from the program card entirely and pass it all in as a component, to make it reusable. 
For example, also export the NextLabel as its own component. Pass in everything within the `<HeadingContentWrapper>` as a heading prop instead.

I'm happy with this PR being an incremental improvement and we can raise another issue to discuss refactoring further.

Reply:
I think that would be a good shout, and can write up a ticket for that.